pom.xml

```diff
@@ -93,11 +93,12 @@
         <version.commons-math3>3.6.1</version.commons-math3><!-- 3.6.1 is approved in CQ22025 -->
         <version.commons-text>1.6</version.commons-text><!-- 1.6 is approved in CQ19028 -->
         <version.findbugs-annotations>1.3.9-1</version.findbugs-annotations><!-- 1.3.9-1 is approved in CQ15081 -->
-        <version.graphhopper>0.13.0</version.graphhopper><!-- 0.13.0 is approved in CQ22337 (api) and CQ22338 (core) -->
+        <version.graphhopper>8.0</version.graphhopper><!-- TODO needs approval -->
```

Comment:
I added issues accordingly:

https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11407
https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11408

Reply:
Have been approved already. Added in pom.xml